06_create_cluster.sh

```diff
@@ -123,6 +123,20 @@ if [[ ! -z "${ENABLE_VIRTUAL_MEDIA_VIA_EXTERNAL_NETWORK}" ]]; then
     oc patch provisioning provisioning-configuration --type merge -p "{\"spec\":{\"virtualMediaViaExternalNetwork\":true}}"
 fi
 
+if [[ -n "${ENABLE_CAPI_E2E}" ]]; then
```

Comment:
Could it be a small separate script that is invoked in the new job?

Reply:
Done

Reply:
... now I'm wondering how we will call it from the job: https://github.com/openshift/release/pull/58839

Reply:
I added the env var back.